src/components/Scheduling/ConfirmationStyles.js

```diff
@@ -0,0 +1,75 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '1rem 2.5rem',
+    height: '90vh',
+    overflowY: 'auto',
+  },
+  container: {
+    margin: '0.7rem 0',
+  },
+  subheadingContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  subheading: {
+    fontweight: 500,
+    color: '#595959',
+    letterSpacing: '1px',
+  },
+  editButton: {
+    fontSize: '18px',
+    color: '#249BE5',
```

Comment:
Are the hard coded colors (like `color: '#249BE5'`, and others in this file) not available in the theme?

Reply:
Just realized that `#249BE5` is in the theme. I'll fix that shortly

All of the other colors are not in the theme file however

- `#595959`
- `#2F2F2F`